[Tierras Quebradas] New Sheet: Tierras Quebradas Edición Revisada

pconcepcion opened the pull request:
## Add a new version of the Tierras Quebradas Sheet to include the changes on the reviewed edition. 

- [x] The pull request title clearly contains the name of the sheet I am editing.
- [x] The pull request title clearly states the type of change I am submitting (New Sheet/New Feature/Bugfix/etc.).

## Pull Request Content

- [x] The pull request makes changes to files in only one sub-folder.
- [x] The pull request does not contain changes to any json files in the translations folder (translation.json is permitted)
- [x] The pull request does not, _without express prior permission from affected parties_, include any material that could be considered to infringe on a Publisher's intellectual property rights, such as logos, images, rules text or other rules content.

## New Sheet Details

Please complete this section if you are adding a new sheet to the repository.

If you are submitting a new sheet to the repository, please fill in any empty spaces indicated by [  ].

- The name of this game is: Tierras Quebradas (Ed. Revisada)
  > _(e.g. Dungeons & Dragons 5th Edition, The Dresden Files RPG)_
- The publisher of this game is: Ediciones T&T  
  > _(e.g. Wizards of the Coast, Evil Hat)_
- The name of this game system/family is: Tierras Quebradas
  > _(e.g. Dungeons & Dragons, FATE)_
- The rules for this game can be found at the following location: https://edicionestyt.com/products/tierras-quebradas-aventuras-entre-la-ley-y-el-caos
  > _(e.g. a link to purchase, ISBN, Half-Price Books)_

- [ ] I have followed the [Character Sheets Standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) when building this sheet.
- [ ] I have authorization from the game's publisher to make this an official sheet on Roll20 with their name attached.
I've done this sheet in collaboration with the author of the game.

Select EXACTLY One:
- [x] This game is a traditionally published game.
- [ ] This game is not a traditionally published game.
- [ ] This sheet is for an unofficial fan game, modification to an existing game, or a homebrew system. [  ADD DETAILS HERE  ]

# Changes / Description

This is an update on the previous character sheet created by elfonochasis for the first version of the game, this sheet is updated to be used with the new Revised edition of the rules.

roll20deploy commented:
[Character Sheet Info](https://app.roll20.net/managesheets/Tierras_Quebradas_edicion_revisada/roll20-character-sheets) *Roll20 Internal Use only.*

kfroll20 commented:
Hello @pconcepcion !

Thank you for your contribution to the community sheets repo! Just a couple things before we move forward with publishing your sheet:
- Your `sheet.json` is missing the preview image and instructions
- This PR is marked as a draft. If you are ready for us to merge/publish the sheet, please mark is as "Ready for review"
<img width="962" height="273" alt="image" src="https://github.com/user-attachments/assets/dd0a1c49-d6ee-4e12-a523-3ea3525e7ffc" />

- Your name of the game should probably include something like "Revised" to help distinguish it from the already existing sheet found here 

<img width="475" height="88" alt="image" src="https://github.com/user-attachments/assets/118f555b-d90e-4ecd-a051-874411e02051" />


Thank you!

pconcepcion commented:
Hi @kfroll20! 
Thanks for your comments!  

I've added the preview and the instructions and modified the comment on the merge request to reflect that the game is _Tierras Quebradas (Ed. Revisada)_ so the reviewed edtion of the game. I don't know if I need to change the name in any other place... 

I think this is ready to review, so unless you tell me otherwise I'll wait a bit and mark it as _Ready for review_
